nekoyume/Assets/_Scripts/UI/Widget/WorldMap.cs

```diff
@@ -418,4 +430,4 @@ private void SetWorldOpenCostTextColor(FungibleAssetValue crystal)
             }
         }
     }
-}
+}
```

Comment:
We need an empty line at the end of file. Could you edit your convention settings?
```suggestion
}

```